src/devices/MemoryLCD/Utility.cs

```diff
@@ -0,0 +1,276 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+
+namespace Iot.Device.MemoryLcd
+{
+    internal static class Utility
+    {
+        public static byte GetAgByte(int b)
+        {
+            return s_byteTable[b];
```

Comment:
The AG byte here is reverse of row number byte, I'm not sure if it's better to use a byte dictionary or directly calculate it

Reply:
I would not use a dictionary for a good reason: your binding can perfectly work on nanoFrmaework which doesn't support dictionaries :-) So just maybe do the math or just have a static table, that would do the trick as well. In terms of size, the math will be better, in terms of performance, the static table would be better. You don't even need a dictionary for that